apps/console/src/extensions/components/logs/components/infinite-scroll-container.tsx

```diff
@@ -16,16 +16,25 @@
  * under the License.
  */
 
+import Modal from "@mui/material/Modal";
```

Comment:
Let's import Modal from oxygen-ui instead

Reply:
We don't have it already in Oxygen.